### DISCUSSION TITLE
AzuraCast + Oracle Cloud Free Tier (Cannot connect as a Streamer of via SFTP)

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#7092] [Q&A]
Hello!

Does anyone have any ideea on how i can make this work?

I have installed Azuracast on an Oracle Cloud VPS, using their Free Tier plan. But i am having these problems, cannot connect to the server as a Live Streamer or via Media SFTP Account. I know this sounds more like an Oracle Cloud question (since there is a free plan, i expect that there are some limitations), but i was hoping that someone on this community will know better if Azuracast is fully working on such a plan or configuration. The rest of the features are working fine, listeners can connect, dashboard is reachable, except these connections attepmpts via SFTP and Live Streaming. 

So are there some settings i should change in Azuracast Configuration or Oracle Cloud Dashboard? 

Lots of thanks!

**Comment by gAlleb** [comment]
It could be a problem with firewall: either internal or external (in oracle dashboard or something). It can happen that only default web ports are open. Maybe try looking there and allow SFTP and Live Streaming ports.

Can you connect directly to `http://your-radio.com:8000/radio.mp3` (example, change to your icecast port number and mountname) ?

**Comment by unudelaradio** [comment]
You are right! 

It only works like this: https://domain.com/listen/radio_name/radio.mp3
The port link doesn't work: https://domain.com:8000/radio.mp3

So i should make some changes in my Oracle Cloud Dashboard? Or in the Ubuntu terminal?

Thank you!

**Reply by gAlleb** [reply]
If firewall is enabled in your Ubuntu you should allow the needed ports:

```
sudo ufw allow 8000/tcp
sudo ufw allow 8005/tcp
sudo ufw allow 2022/tcp
sudo ufw allow 'OpenSSH'
etc...
```
You can check with `sudo ufw status` to see if it's enabled or disabled. If it's disabled I highly recommend to enable it `after allowing ports` and finding the cause of your issue.

But since you can connect to ssh to your server - probably `ssh/sftp` (not to confuse OpenSSH sftp (port 22) with Azuracast sftp (port 2022)) ports are already enabled or `ufw` is disabled. 

So next you must look into Oracle dashboard for its External Firewall and allow necessary ports there. Probably they allow only default web ports (80 and 443) and basic ssh/sftp ports from the start.

**Comment by unudelaradio** [comment]
I'll take a look and follow the steps. Many thanks for your help!
